src/Livewire/MyProfileComponent.php

```diff
@@ -24,4 +24,18 @@ public function render()
         return view($this->view);
     }
 
+    public static function canView(): bool
+    {
+        return true;
+    }
+
+    public static function sort(): int
```

Comment:
i think this should be `getSort`

Reply:
@boris-glumpler I agree with this, could you update to `getSort` to match Filament?

Reply:
Done!